### ISSUE TITLE
CORS Issue when trying to print labels from our Application hosted on Azure App Services

### ISSUE THREAD

**Issue by kwokmanDiagnostics** [dymosoftware/DCD-SDK-Sample#41]
Hello,

Our application is currently hosted on Azure App Services and then we are using dymo js library. We are facing challenges with CORS issues. May I know if anyone has figured out a way around it. I appreciate it! Thank you!

**Comment by malcolm-mcclintock** [comment]
bump, also looking to figure out CORS workaround!

**Comment by kwokmanDiagnostics** [comment]
Yeah, CORS issue is a huge blocker for us. We couldn’t print directly from our application running on Azure App Services. We currently are downloading .label file and printing it manually. Even we tried setting up proxy and still not working. Any help would be appreciated and paid. Thank you!

**Comment by rjCousineau** [comment]
Having the same issue, setting header `Access-Control-Allow-Origin: *` doesn't solve the issue.  What to do?

**Comment by kwokmanDiagnostics** [comment]
@rjCousineau  Still no luck! We are downloading .label files and printing them individually. Any update on this folks?
